### ISSUE TITLE
"All fields" dropdown needs to specific foreground color

### ISSUE THREAD

**Issue by EdwinGuzman** [NYPL/design-toolkit#194]
I'm getting an error in aXe about this.

![screen shot 2017-07-31 at 11 37 22 am](https://user-images.githubusercontent.com/1280564/28785537-bb09b1be-75e4-11e7-93e4-cd1e2d8dfceb.png)

@wlla , this error is coming up in Discovery but should be fixed here.
